.github/workflows/tox-linters.yml

```diff
@@ -18,13 +18,13 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-        - 3.8
-        - 3.7
+        # - 3.8
+        # - 3.7
         - 3.6
```

Comment:
I prefer having 3.8 instead and just remove the lines, no need to comment them out anymore.